### ISSUE TITLE
arch build failes because of daita/wireguard? 

### ISSUE THREAD

**Issue by xLogiiCx** [mullvad/mullvadvpn-app#7079]
### Is it a bug?

- [X] I know this is an issue with the app, and contacting Mullvad support is not relevant.

### I have checked if others have reported this already

- [X] I have checked the issue tracker to see if others have reported similar issues.

### Current Behavior

when building the wireguard app on arch linux using the aur the build failes for a custom build. 

### Expected Behavior

the build succseeds

### Steps to Reproduce

try to build the mullvad-vpn-cli app using the aur

### Failure Logs

```shell
error: failed to run custom build command for `wireguard-go-rs v0.0.0 (/tmp/makepkg/mullvad-vpn-cli/src/mullvadvpn-app/wireguard-go-rs)`

Caused by:
  process didn't exit successfully: `/tmp/makepkg/mullvad-vpn-cli/src/mullvadvpn-app/target/release/build/wireguard-go-rs-d40f5bfc91516353/build-script-build` (exit status: 1)
  --- stdout
  cargo::rustc-check-cfg=cfg(daita)
  cargo::rerun-if-changed=libwg

  --- stderr
  Error from cd "./libwg" && CGO_ENABLED="1" GOARCH="amd64" GOOS="linux" "go" "build" "-v" "-o" "/tmp/makepkg/mullvad-vpn-cli/src/mullvadvpn-app/target/release/build/wireguard-go-rs-ad78211061f55ed7/out/libwg.a" "-buildmode" "c-archive" "--tags" "daita"

  stdout:



  -------
  stderr:

  go: downloading golang.org/x/sys v0.19.0
  go: downloading golang.org/x/net v0.24.0
  go: downloading golang.org/x/crypto v0.22.0
  runtime/cgo
  net
  golang.zx2c4.com/wireguard/device
  # golang.zx2c4.com/wireguard/device
  wireguard-go/device/daita.go:287:21: could not determine kind of name for C.MaybenotAction_InjectPadding
  wireguard-go/device/daita.go:292:22: could not determine kind of name for C.MaybenotAction_InjectPadding_Body


  -------
  Error: Command exited with a non-zero exit code: exit status: 1

  Caused by:
      Failed to execute command: cd "./libwg" && CGO_ENABLED="1" GOARCH="amd64" GOOS="linux" "go" "build" "-v" "-o" "/tmp/makepkg/mullvad-vpn-cli/src/mullvadvpn-app/target/release/build/wireguard-go-rs-ad78211061f55ed7/out/libwg.a" "-buildmode" "c-archive" "--tags" "daita"
```
```


### Operating system version

6.11.5-zen1-1-zen

### Mullvad VPN app version

not installed yet
trying to install mullvad-vpn-cli-2024.6-1

### Additional Information

_No response_

**Comment by MarkusPettersson98** [comment]
Hello, thanks for reaching out, but you should try to contact the author of the AUR package instead. They should make sure to recursively checkout the submodules that `wireguard-go-rs` depend on.

**Comment by khairizalkhalid** [comment]
The maintainer fixed the PKGBUILD in version 2024.6-2 which is now uses a static commit for the maybenot dependency.

**Comment by MarkusPettersson98** [comment]
@khairizalkhalid Can we close this issue then? :blush:

**Comment by khairizalkhalid** [comment]
@MarkusPettersson98 maybeyeah :)

**Comment by MarkusPettersson98** [comment]
Okay, closing this issue since we don't have the ownership of the AUR package anyway. If there would be an issue with our build chain, the maintainer is of course free to contact us.

**Comment by xLogiiCx** [comment]
> The maintainer fixed the PKGBUILD in version 2024.6-2 which is now uses a static commit for the maybenot dependency.

This fixed it thank you.
